modules/testModule.py

```diff
@@ -1,138 +1,285 @@
-import re
 from asyncio import sleep
-from utilities import get_question_id, is_test_response
-from modules.module import Module, Response
+import re
+from textwrap import dedent
+from typing import cast
+
 from jellyfish import jaro_winkler_similarity
-from config import TEST_QUESTION_PREFIX, TEST_RESPONSE_PREFIX, test_response_message
+
+from config import TEST_MESSAGE_PREFIX, TEST_RESPONSE_PREFIX, test_response_message
+from modules.module import IntegrationTest, Module, Response
 from servicemodules.serviceConstants import Services
+from utilities import get_question_id, is_test_response
+from utilities.serviceutils import ServiceMessage
+from utilities.utilities import is_bot_dev
 
 
 class TestModule(Module):
-    # This module is the only module that gets stampy to ask its self multiple questions
-    # In test mode, stampy only responds to itself, whereas in other modes stampy responds only to not itself
-    TEST_PREFIXES = {TEST_QUESTION_PREFIX, TEST_RESPONSE_PREFIX}
-    TEST_MODULE_PROMPTS = {"test yourself", "test modules"}
+    """
+    This module is the only module that gets stampy to ask himself multiple questions.
+    In test mode, stampy only responds to himself, whereas in other modes stampy responds only to not himself.
+    
+    ### How to test Stampy
+    
+    - You can only test Stampy in the #talk-to-stampy channel
+    - You can test Stampy in one of three ways
+        1. test all modules; `s, test yourself` or `s, test modules`
+        2. test a subset of modules `s, test modules <module-name-1> <module-name-2> <module-name-3>
+        3. test one module: `s, test module <module-name>`
+    """
+
+    TEST_PREFIXES = {TEST_MESSAGE_PREFIX, TEST_RESPONSE_PREFIX}
+    TEST_MODULE_PROMPTS = {"test yourself", "test modules", "test module"}
     TEST_PHRASES = {TEST_RESPONSE_PREFIX} | TEST_MODULE_PROMPTS
-    TEST_MODE_RESPONSE_MESSAGE = (
-        "I am running my integration test right now and I cannot handle your request until I am finished"
-    )
+    TEST_MODE_RESPONSE_MESSAGE = "I am running my integration test right now and I cannot handle your request until I am finished"
```

Comment:
do we have maximum character width? this line might be 'too long'

Reply:
IIRC we use black with standard settings for autoformatting but